scour/scour.py

```diff
@@ -3225,6 +3233,10 @@ def format_usage(self, usage):
    help="don't remove IDs starting with the given prefix")
 _options_parser.add_option_group(_option_group_ids)
 
+_option_group_compatibility = optparse.OptionGroup(_options_parser, "SVG compatibility checks")
+_option_group_compatibility.add_option("--error-on-flowtext",
+   action="store_true", dest="error_on_flowtext", default=False,
+   help="In case the input SVG uses flow text, bail out with error. Otherwise only warn. (default: False)")
```

Comment:
I think you're missing an `_options_parser.add_option_group(_option_group_compatibility)` here...